tasmota/xdrv_01_webserver.ino

```diff
@@ -921,6 +921,10 @@ void WSContentSend_Temp(const char *types, float f_temperature) {
   WSContentSend_PD(HTTP_SNS_F_TEMP, types, Settings->flag2.temperature_resolution, &f_temperature, TempUnit());
 }
 
+void WSContentSend_Pressure(const char *types, float f_pressure) {
+  WSContentSend_PD(HTTP_SNS_F_PRESSURE, types, Settings->flag2.pressure_resolution, &f_pressure);
+}
+
```

Comment:
Don't add it here. Move it into your driver. As there are several other pressure drivers look how they do it.

Also have a look at their "SeaPressure" functionality. A free bonus if you manage to integrate your driver like the other ondes did.